tests/NCronJob.Tests/MockHostApplicationLifetime.cs

```diff
@@ -1,14 +1,45 @@
+using Microsoft.Extensions.DependencyInjection;
 using Microsoft.Extensions.Hosting;
 
 namespace NCronJob.Tests;
 
-public class MockHostApplicationLifetime : IHostApplicationLifetime
+public class MockHostApplicationLifetime : IHostApplicationLifetime, IDisposable
```

Comment:
You can make that `sealed` to save some boiler-plate code (like `GC.SuppressFinalize`)

Reply:
done